js/utils.js

```diff
@@ -236,7 +239,7 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
         context = undefined;
       }
 
-      return getMomentDate(context).format(block.hash.format || 'DD MMMM YYYY');
+      return getMomentDate(context).format(moment()._locale._longDateFormat.ll || moment()._locale._longDateFormat.LL);
```

Comment:
1. `moment()._locale._longDateFormat.ll` should be the same as `moment.localeData().longDateFormat('ll')`. Use the function instead of accessing the `_longDateFormat` object directly.
2. Why are you using `ll` with the `LL` fallback? Why not just reference `LOCALE_FORMATS.DATE`?
3. `block.hash.format` needs to be kept because it gives users a way to customise the output format. If you remove `block.hash.format` then existing Handlebars helpers that use a custom format will break.